.env.production

```diff
@@ -1,4 +1,4 @@
-REACT_APP_AGGREGATOR_API=https://aggregator-api.kyberswap.com
+REACT_APP_AGGREGATOR_API=https://meta-aggregator-api.kyberswap.com
```

Comment:
404 so far

Reply:
try this 
https://meta-aggregator-api.kyberswap.com/polygon/api/v1/routes?tokenIn=0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE&tokenOut=0xc2132D05D31c914a87C6611C10748AEb04B58e8F&amountIn=1000000000000000000&saveGas=false&includedSources=uniswapv3%2Ckyberswapv2%2Ckyberswap%2Ckyberswap-static&gasInclude=true